### DISCUSSION TITLE
Older versions past 1.8? (Solved)

### DISCUSSION THREAD

**Discussion by Deoxist** [Jikoo/OpenInv#58] [Q&A] [answered]
Are the versions past 1.8 public/available to download? I have an idea for a server that uses older versions and would need a plugin like this one.

**Comment by Jikoo** [comment, marked as answer]
Historic versions can be found on Bukkit or in lishid's repo's releases. You should be linked the correct version for your server when loading on an unsupported version. Alternately, you can look it up yourself [in the code](https://github.com/Jikoo/OpenInv/blob/6a31015ee2a4991293b5533428455a8a51e4ed49/plugin/src/main/java/com/lishid/openinv/util/InternalAccessor.java#L51).

Note that outdated versions may require outdated Java to run properly.
